docs/samples/browser-plugin-meetings/app.js

```diff
@@ -2811,6 +2818,12 @@ const participantsList = document.querySelector('#participant-list');
 const participantTable = document.querySelector('#participant-table');
 const participantButtons = document.querySelector('#participant-btn');
 
+participantTable.addEventListener('click', (event) => {
+  const selectedParticipant = meeting.members.membersCollection.get(event.target.id);
```

Comment:
- We should add `event.preventDefault()` at the beginning and `event.stopPropagation()` at the end.
- We should add a check `if (!selectedParticipant) { return; }`

Reply:
- Table click doesn't have an action by default that needs prevention (@rarajes2)
- We need that check as well as a check to do anything only if the radio button is clicked. This will keep calling the membersCollection.get when clicked anywhere in the table any number of times

Reply:
Correct @mkesavan13, We don't need preventDefault() but `stopPropagation()` would stop the further bubbling.

Reply:
Again, we know the table won't have another click event listener. So stopPropogation doesn't seem mandatory to me.